Deep_Learning/Dog Breeds Classification/web-app/backend/check.py

```
@@ -0,0 +1,45 @@
+# import tensorflow
+from tensorflow.keras.models import load_model
+import numpy as np
+from tensorflow.keras.preprocessing import image
+import tensorflow as tf
```

Ruff (0.8.2)

`tensorflow` imported but unused

Remove unused import: `tensorflow`

(F401)